### PR TITLE
project evalautions now have a GUID to be linked to emails + some bug…

### PR DESCRIPTION
**Pull request by ekm6**
… fixes completed

### CODE REVIEW

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs [hunk 1]
```diff
@@ -116,7 +114,8 @@ public async Task<ActionResult> DeleteProject(int id)
         [HttpPost("save")]
         public async Task<ActionResult> SaveProject([FromBody] ProjectUpdateRequest project)
         {
-            int userId = Int32.Parse(HttpContext.User.Identity.Name);
+            //    int userId = Int32.Parse(HttpContext.User.Identity.Name);
+            int userId = 1;
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs:118]: I assume you plan to figure out how to get the user id??

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/ProjectUpdateRequest.cs [hunk 2]
```diff
@@ -39,7 +39,7 @@ public class LinkRequest
 
     public class EvaluationRequest
     {
-        public int RoleId { get; set; }
+        public string Role { get; set; }
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/ProjectUpdateRequest.cs:42]: What is the purpose of this change? Shouldn't we accept RoleId?

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs (hunk 1 above)
Comment (ekm6) [reply Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/ProjectsController.cs:118]: I set it equal to 1 for testing purposes, I'll change it back

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/ProjectUpdateRequest.cs (hunk 2 above)
Comment (ekm6) [reply Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/ProjectUpdateRequest.cs:42]: I thought we moved away from using the the role table for projects. Instead, there is a "projectRole" attribute on project evaluations

Comment (joeldevries) [reply Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/ProjectUpdateRequest.cs:42]: Ah, gotcha, this is good then.
